overlord/state/taskrunner.go

```diff
@@ -69,9 +69,8 @@ func (r *TaskRunner) Handlers() map[string]HandlerFunc {
 	return r.handlers
 }
 
-// taskFail marks task t and all tasks waiting (directly and indirectly on it) as in ErrorStatus.
-func taskFail(task *Task) {
-	task.SetStatus(ErrorStatus)
+// propagateError sets to ErrorStatus all tasks directly and indirectly waiting on t.
```

Comment:
Hmmm.. I went to do the change, but while doing so the phrasing felt slightly strange to read. It's "setting the task status to ErrorStatus", not "setting ErroStatus to the tasks status".